membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -198,7 +198,8 @@ class AccountController(commonActions: CommonActions, override val controllerCom
       accountHasMissedRecentPayments = false,
       safeToUpdatePaymentMethod = true,
       isAutoRenew = isAutoRenew,
-      alertText = alertText
+      alertText = alertText,
+      accountId = accountSummary.id.get
```

Comment:
This is for some reason defined as `case class AccountId(get: String) extends AnyVal` so that `get` is safe.